spec/features/admin_user/new_item_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.feature "the admin can create new items", type: :feature do
+
+    let(:dessert) { Category.create(name: "dessert") }
+
+    let(:arya) { User.create(full_name: "Arya of House Stark",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.